index.js

```diff
@@ -33,6 +33,7 @@ function setTypeParser (oid, format, parseFn) {
     parseFn = format
     format = 'text'
   }
+  if (oid == null) throw new Error("Missing oid");
```

Comment:
How about explicitly checking that oid is an integer?

```js
if (!Number.isInteger(oid)) {
    throw new Error('oid must be an integer: ' + oid)
}
```

Reply:
Sure ! Meanwhile i fixed style and added a basic test.

Reply:
Could use `TypeError` rather than a vanilla Error too.